website/content/docs/plugins/hcp-support.mdx

```diff
@@ -0,0 +1,114 @@
+---
+description: |
+  The HCP Packer support allows plugins to be used with HCP and push metadata to the HCP Packer registry.
+page_title: Integration Program
+---
+
+# HCP Packer Support
+
+~> **Note:** HCP Packer is under active development, and we suggest plugin maintainers to keep up with the SDK changes
+for more on HCP Packer support.
+
+~> **Note:** If you're looking to develop a new plugin, please refer first to the [developing
+plugins](/docs/plugins/creation#developing-plugins) page.
+
+The HCP Packer support consists on making the Image metadata available to Packer through the builder artifact. Before pushing
+metadata to the HCP Packer registry, Packer will query for the Image metadata by looking for a [Image](https://github.com/hashicorp/packer-plugin-sdk/blob/b688c9db722f8e9685310b03ebcc55bfa6b7a674/packer/registry/image/image.go#L22)
+under a [key](https://github.com/hashicorp/packer-plugin-sdk/blob/b688c9db722f8e9685310b03ebcc55bfa6b7a674/packer/registry/image/image.go#L15)
+in the artifact.
```

Comment:
I would suggest linking out to the module docs so that users can see the types and basic examples on how to use the ArtifactOverrideFunc type. The latest module docs can be found at https://pkg.go.dev/github.com/hashicorp/packer-plugin-sdk/packer/registry/image

Reply:
Ohh good idea! I haven't thought about that but that's the correct thing to point to. Thanks